packages/@aws-cdk/aws-bedrock-agentcore-alpha/agentcore/runtime/perms.ts

```diff
@@ -101,6 +101,16 @@ export const RUNTIME_WORKLOAD_IDENTITY_ACTIONS = [
   'bedrock-agentcore:GetWorkloadAccessTokenForJWT',
   'bedrock-agentcore:GetWorkloadAccessTokenForUserId',
 ];
+
+/**
+ * Bedrock model invocation permissions
+ * Used by the runtime to invoke Bedrock models
+ */
+export const RUNTIME_BEDROCK_MODEL_INVOCATION_ACTIONS = [
+  'bedrock:InvokeModel',
+  'bedrock:InvokeModelWithResponseStream',
```

Comment:
I understand that this is what is written in the actual documentation.
https://docs.aws.amazon.com/bedrock-agentcore/latest/devguide/runtime-permissions.html

However, Converse (Stream) is a different natural approach for users to invoke the model. I believe it should be permitted by default.
```suggestion
  'bedrock:InvokeModelWithResponseStream',
  'bedrock:Converse',
  'bedrock:ConverseStream',
];
```

Reply:
It seems I misunderstood and thought an Action-level operation was required. I have also checked the documentation and confirmed that it is okay. I will close this comment.